client/landing/jetpack-cloud/components/date-picker/index.jsx

```diff
@@ -85,7 +85,7 @@ class DatePicker extends Component {
 	};
 
 	goToActivityLog = () => {
-		page.redirect( `/activity/${ this.props.siteSlug }` );
+		page.redirect( `/backups/activity/${ this.props.siteSlug }` );
```

Comment:
`backupActivityPath` is always an option for cases like this!

Reply:
I considered that, forget what stopped me from doing it.